posts/2017-10-15-creating-a-serverless-graphql-to-rest-gateway-on-top-of-the-flickr-api.md

```diff
@@ -370,11 +395,17 @@ module.exports = {
   ]
 }
 ```
-I specifically include the webpack config for this project here to note the usage of webpack's [provide plugin](https://webpack.js.org/plugins/provide-plugin/), which will be relevant to the rest of this article. What this does is it allows you to call exports from node modules without having to explicitly import them in the files in which you use them. As a matter of personal preference, I enjoy that this plugin saves me from having to remember to import frequently used things such as the built-in GraphQL scalar types, which we'll be using often throughout this project. It can get difficult to remember which types you are using in large type definitions, which can cause confusion during development from frequent undefined errors. So going forward **if you are confused why you're seeing things like `GqlString` instead of `GraphQLString`, this is why.**
+There are three important things to note here.
+
+One: I specifically include the webpack config for this project highlight webpack's [provide plugin](https://webpack.js.org/plugins/provide-plugin/). It allows you to call exports from node modules without having to explicitly import them in the files in which you use them.
```

Comment:
I'm not sure if I should update this section, because I've since moved everything from provide plugin to a babel plugin instead. The reason being that it was necessary in order to get tests running properly. It may not matter for the context of this tutorial though.